imagenet/README.md

```diff
@@ -31,10 +31,16 @@ ImageNet dataset is large and time-consuming to download. To get started quickly
 python main.py -a resnet18 --dummy
 ```
 
+To test XPU
```

Comment:
This does not actually bring in value. Users will see the flag described below and use it if needed. I suggest to drop.